internal/translator/translate.go

```diff
@@ -203,6 +214,23 @@ func translateRoutes(irRoutes []irv1beta1.Route, routeLCP string) ([]hpv1.Route,
 	return routes, includes, warnings
 }
 
+func translateTCPProxy(irTCPProxy *irv1beta1.TCPProxy) *hpv1.TCPProxy {
+
+	proxy := &hpv1.TCPProxy{}
+
+	for _, irService := range irTCPProxy.Services {
+
+		hpService := hpv1.Service{
+			Name:   irService.Name,
+			Port:   irService.Port,
+			Weight: irService.Weight,
```

Comment:
What about the other service fields? Healthcheck, Strategy, UpstreamValidation? Could just hoist the code from `translateRoute` into something that translates a service and does all the fields ...

Reply:
Those actually don't do anything for TCPProxy, afaict, so I left them out. I can do it that way if you think it's better though.

Reply:
Now I think about it, I'll just do that, regardless of whether it does anything. Thanks.